src/github.com/matrix-org/dendrite/cmd/mediaapi-integration-tests/main.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 Vector Creations Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	 http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"sync"
+)
+
+const origin = "matrix.org"
+const mediaID = "rZOBfBHnuOoyqBKUIHAaSbcM"
```

Comment:
It is. I'm working on the integration tests to be more like the syncserver tests. It will upload a file and then download that file when I'm done.